src/vm/mscorlib.h

```diff
@@ -734,6 +737,17 @@ DEFINE_CLASS(RAW_SZARRAY_DATA,      CompilerServices,       RawSzArrayData)
 DEFINE_FIELD(RAW_SZARRAY_DATA,      COUNT,                  Count)
 DEFINE_FIELD(RAW_SZARRAY_DATA,      DATA,                   Data)
 
+DEFINE_CLASS(TAIL_CALL_FRAME,       CompilerServices,       TailCallFrame)
+DEFINE_FIELD(TAIL_CALL_FRAME,       PREV,                   Prev)
+DEFINE_FIELD(TAIL_CALL_FRAME,       TAILCALL_AWARE_RETURN_ADDRESS, TailCallAwareReturnAddress)
+DEFINE_FIELD(TAIL_CALL_FRAME,       NEXT_CALL,              NextCall)
```

Comment:
Could you please also add `*_U` version of the macros here for the type and fields? It will guarantee with runtime asserts that the managed and unmanaged layouts are the same.